### PR TITLE
mobile styling

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: src/components/developers/developers.component.jsx [hunk 1]
```diff
@@ -66,19 +68,23 @@ const Developers = () => {
...
+            <div className={classes.section3SubTitle}>
+              <Title>Community centric protocol</Title>
+            </div>
             <p className={classes.paragraphMargin}>One of the most important things about Hermez is the way it decides who the next block creator should be.</p>
             <p className={classes.paragraphMargin}>In a sentence, block creators are selected via a burn auction, except rather than burning tokens, a 40% of the winning bid goes back to be reinvested in Ethereum public goods through Gitcoin quadratic funding grants.</p>
             <p className={classes.paragraphMargin}>We call this mechanism proof-of-donation.</p>
             <TextLink to='https://docs.hermez.io/#/' text='More in documentation' />
           </div>
-          <div className={classes.col2}>
+          <div className={`${classes.col2} ${classes.webOnly}`}>
```
Comment (AlbertoElias) [inline src/components/developers/developers.component.jsx]: web => desktop as all layouts are Web layouts

File: src/components/footer/footer.component.jsx [hunk 2]
```diff
@@ -11,23 +11,25 @@ export const Footer = ({ routes }) => {
...
-          <div>
+          <div className={classes.col3}>
             <ul className={classes.list}>
               <li className={`${classes.listItem} ${classes.listItemHead}`}>
                 Hermez
               </li>
               <li className={classes.listItem}>
                 <Link to={routes.find(el => el.id === 2).to}>
-                  {routes.find(el => el.label === 'Payments network').label}
+                  {routes.find(el => el.id === 2).label}
```
Comment (AlbertoElias) [inline src/components/footer/footer.component.jsx:54]: Maybe a comment or a const to explain where the 2 comes from

File: src/components/header/header.component.jsx [hunk 3]
```diff
@@ -17,14 +18,9 @@ export const Header = ({ routes, onOpenMobileMenu }) => {
...
               <li key={index} className={classes.linkWrapper}>
-                <Link
-                  className={classes.link}
-                  to={route.to}
-                >
-                  {route.label === 'Hermez' ? (
-                    <Logo />
-                  ) : route.label}
-                </Link>
+                {route.id === 1
```
Comment (AlbertoElias) [inline src/components/header/header.component.jsx:43]: same with the id here

File: src/components/payments-network/payments-network.styles.js [hunk 4]
```diff
@@ -1,95 +1,148 @@
...
+    }
+  },
+  buttonWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    [theme.breakpoints.sm]: {
+      justifyContent: 'left'
     }
   },
+  section2: {
```
Comment (AlbertoElias) [inline src/components/payments-network/payments-network.styles.js]: If it's empty maybe you could remove it?

**Review by AlbertoElias — CHANGES_REQUESTED** [review]
A few small comments

File: src/components/developers/developers.component.jsx (hunk 1 above)
Comment (mmilenkovic) [reply src/components/developers/developers.component.jsx]: Makes sense. Changed.

File: src/components/footer/footer.component.jsx (hunk 2 above)
Comment (mmilenkovic) [reply src/components/footer/footer.component.jsx:54]: Those are being passed from layout component. Comment added.

File: src/components/header/header.component.jsx (hunk 3 above)
Comment (mmilenkovic) [reply src/components/header/header.component.jsx:43]: Those are being passed from layout component. Comment added.

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**